hid-ft260.c

```diff
@@ -628,6 +631,13 @@ static int ft260_i2c_xfer(struct i2c_adapter *adapter, struct i2c_msg *msgs,
 		return ret;
 	}
 
+	if (time_is_before_jiffies(dev->need_wakeup_at)) {
+		ft260_dbg("sending status request to wakeup device");
+		ret = ft260_xfer_status(dev);
```

Comment:
Would it be better to exchange the order of these two lines to issue the `ft260_xfer_status` request earlier?

Reply:
fixed